icu4c/source/i18n/usearch.cpp

```diff
@@ -48,11 +48,11 @@ static const Normalizer2Impl *g_nfcImpl = NULL;
 */
 static
 inline void setColEIterOffset(UCollationElements *elems,
-                      int32_t             offset)
+                              int32_t offset,
+                              UErrorCode& status)
```

Comment:
optional: I have a preference for `UErrorCode &status` with the ampersand adjacent to the parameter name, like how C types were intended and how the code here uses. But I understand that that's falling out of vogue.

Reply:
Changed to use `UErrorCode &status`.